play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -229,7 +229,11 @@ class UserOptionsPanel extends PSRoomPanel {
 	getTargets() {
 		const [, targetUser, targetRoomid] = this.props.room.id.split('-');
 		let targetRoom = (PS.rooms[targetRoomid] || null) as ChatRoom | null;
-		if (targetRoom?.type !== 'chat') targetRoom = null;
+		if (targetRoom?.type !== 'chat') {
+			targetRoom = targetRoom?.getParent()?.type === 'chat' ?
+				targetRoom?.getParent() as ChatRoom :
+				null;
+		}
```

Comment:
```suggestion
		if (targetRoom?.type !== 'chat') targetRoom = targetRoom?.getParent();
		if (targetRoom?.type !== 'chat') targetRoom = targetRoom?.getParent();
		if (targetRoom?.type !== 'chat') targetRoom = null;
```

Maybe this could be merged with the `/inopener` command later...